tuxemon/technique.py

```diff
@@ -309,6 +353,7 @@ def damage(self, user: Monster, target: Monster) -> EffectResult:
             if not hit:
                 damage //= 2
             target.current_hp -= damage
+            success = damage
```

Comment:
this would probably be better as
```python
success = damage > 0
```
in general, passing numeric values around and treating the as booleans can lead to bugs or confusing code later.  better to cast as the correct type as soon as possible

Reply:
ill just delete that, i added it and its not needed anymore anyway